Example/Podfile

```diff
@@ -3,9 +3,9 @@ source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
 def common
-  pod 'RxSwift', '~> 5.0'
-  pod 'RxCocoa', '~> 5.0'
-  pod 'RealmSwift', '~> 4.0'
```

Comment:
These changes are not needed, cause `~>` is specified. If you want use latest versions of dependencies just run `pod update` and it will resolve to `5.1.1` for RxSwift and `4.4` for Realm